### PR TITLE
fix: Fixed disaplying quantity of user's items

### PR DESCRIPTION
**Pull request by TheSubliminal**

### CODE REVIEW

File: client/src/components/DashboardHeader/index.jsx [hunk 1]
```diff
@@ -40,27 +40,27 @@ const DashboardHeader = ({
         </li>
         <li>
           <Link to={`${baseUrl}?tab=repositories`} className={getClassName(tabs.repositories, activeTab)}>
-            Repositories{repoCount && <span>{repoCount}</span>}
+            Repositories{<span>{repoCount || 0}</span>}
```
Comment (alexandrtovmach) [inline client/src/components/DashboardHeader/index.jsx]:
```suggestion
            Repositories <span>{repoCount || 0}</span>
```

Comment (denterekhov) [inline client/src/components/DashboardHeader/index.jsx]: Could we avoid using curly braces around <span> element?

Comment (TheSubliminal) [reply client/src/components/DashboardHeader/index.jsx]: Fixed

Comment (TheSubliminal) [reply client/src/components/DashboardHeader/index.jsx]: Fixed

**Pull request merged**